jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskprocesscontext/TaskProcessContextPresenter.java

```diff
@@ -68,18 +77,23 @@ public TaskProcessContextPresenter(
             PlaceManager placeManager,
             Caller<TaskQueryService> taskQueryService,
             Caller<DataServiceEntryPoint> dataServices,
-            Event<ProcessInstancesWithDetailsRequestEvent> processInstanceSelected
+            Event<ProcessInstancesWithDetailsRequestEvent> processInstanceSelected,
+            ActivityManager activityManager
     ) {
         this.view = view;
         this.taskQueryService = taskQueryService;
         this.dataServices = dataServices;
         this.placeManager = placeManager;
         this.processInstanceSelected = processInstanceSelected;
+        this.activityManager = activityManager;
     }
 
     @PostConstruct
     public void init() {
         view.init(this);
+        final Set<Activity> activity = activityManager.getActivities(new DefaultPlaceRequest(PROCESS_INSTANCE_DETAILS));
+        enableProcessInstanceDetails = activity.isEmpty() == false;
```

Comment:
@manstis, your assumption is correct from what I can tell. The reason this works on kie-drools-wb is because jbpm-console-ng-process-runtime-client module isn't present, so the perspective/activity isn't available at compile and runtime.